policy/lfu/priority_queue.go

```diff
@@ -40,29 +40,29 @@ func newPriorityQueue[K comparable, V any](cap int) *priorityQueue[K, V] {
 // see example of priority queue: https://pkg.go.dev/container/heap
 var _ heap.Interface = (*priorityQueue[struct{}, interface{}])(nil)
 
-func (l priorityQueue[K, V]) Len() int { return len(l) }
+func (q priorityQueue[K, V]) Len() int { return len(q) }
 
-func (l priorityQueue[K, V]) Less(i, j int) bool {
-	if l[i].referenceCount == l[j].referenceCount {
-		return l[i].referencedAt.Before(l[j].referencedAt)
+func (q priorityQueue[K, V]) Less(i, j int) bool {
+	if q[i].referenceCount == q[j].referenceCount {
+		return q[i].referencedAt.Before(q[j].referencedAt)
 	}
-	return l[i].referenceCount < l[j].referenceCount
+	return q[i].referenceCount < q[j].referenceCount
 }
 
-func (l priorityQueue[K, V]) Swap(i, j int) {
-	l[i], l[j] = l[j], l[i]
-	l[i].index = i
-	l[i].index = j
+func (q priorityQueue[K, V]) Swap(i, j int) {
+	q[i], q[j] = q[j], q[i]
+	q[i].index = i
+	q[j].index = j
```

Comment:
@chenyanchen Did you fix just this line?

Reply:
Yes